recipe/meta.yaml

```diff
@@ -13,7 +13,10 @@ source:
 
 build:
     number: 1
-    skip: True  # [py35 and win32]
+    features:
+     - vc9  # [win and py27]
+     - vc10  # [win and py34]
+     - vc14  # [win and py35]
```

Comment:
You also need a build requirement on Python (no version) - so that the features get activated and associated appropriately.  Without it, I don't think the py?? selectors work.